src/test/java/org/apache/commons/imaging/common/bytesource/ByteSourceTest.java

```diff
@@ -66,6 +64,22 @@ protected File createTempFile(final byte src[]) throws IOException {
         }
         final byte longArray[] = (baos.toByteArray());
 
-        return new byte[][] { emptyArray, single, simple, zeroes, longArray, };
+        return new byte[][]{emptyArray, single, simple, zeroes, longArray,};
+    }
+
+    @Test
```

Comment:
Yes, I know.
However, for what reason ever I experienced a lot of people preferring the try/catch notation over the annotation approach.
That's why I write/wrote it this way.
In the hope to have to rewrite as less code in the end as possible.